build/yamls/antrea-windows-ovs-containerd.yml

```diff
@@ -0,0 +1,336 @@
+apiVersion: v1
+data:
+  Install-WindowsCNI-Containerd.ps1: |
+    $ErrorActionPreference = "Stop";
+    mkdir -force c:/var/log/antrea
+    $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
+    $mountPath =  ($mountPath.Replace('\', '/')).TrimEnd('/')
+    mkdir -force C:/var/run/secrets/kubernetes.io/serviceaccount
+    cp $mountPath/var/run/secrets/kubernetes.io/serviceaccount/ca.crt C:/var/run/secrets/kubernetes.io/serviceaccount
+    cp $mountPath/var/run/secrets/kubernetes.io/serviceaccount/token C:/var/run/secrets/kubernetes.io/serviceaccount
+    mkdir -force c:/opt/cni/bin/
+    mkdir -force c:/etc/cni/net.d/
+    cp $mountPath/k/antrea/cni/* c:/opt/cni/bin/
+    cp $mountPath/etc/antrea/antrea-cni.conflist c:/etc/cni/net.d/10-antrea.conflist
+    mkdir -force c:/k/antrea/bin
+    cp $mountPath/k/antrea/bin/antctl.exe c:/k/antrea/bin/antctl.exe
+  Run-AntreaAgent-OVS-Containerd.ps1: |
+    $ErrorActionPreference = "Stop"
+    $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
+    $mountPath =  ($mountPath.Replace('\', '/')).TrimEnd('/')
+    Start-Sleep -Seconds 3
```

Comment:
May I know why sleep 3 second?

Reply:
to ensure that antrea-ovs container updates the path (adds path to openvswitch binaries and windows/system32) and then we can update path for this session in next line. It happens in less than a second but to be on safer side 3 sec is used.

Reply:
I feel waiting for a specified duration may pose a risk for lower performance testbed. How about waiting until the path is updated?

Reply:
This script runs in antrea-agent container to start antrea-agent. During agent initialisation an ovs command is ran. To successfully run ovs command inside antrea-agent container, path of ovs binaries needs to be added in env var PATH here. Now, this antrea-agent container uses image `antrea-windows` which doesn't contain ovs binaries. So, we need path of ovs binaries present in antrea-ovs container. Now absolute path of antrea-ovs container cannot be predicted earlier. When antrea-ovs container starts it adds the absolute path to its ovs binaries and we fetch that updated path in antrea-agent container after 3 secs.

Reply:
If adding the path is necessary, I feel we should at least use the existence of the relevant paths as the condition for waiting, instead of using a fixed sleep time. @tnqn Do you have more suggestions？

Reply:
@Atish-iaf -  if antrea-agent container can use readiness probe with initial delay of say 3 seconds
readinessProbe:
        initialDelaySeconds:  3 seconds
        periodSeconds: 
and antrea-ovs container can use lifecycle/poststart to make sure it sets the path and then only antrea-agent container starts ?
lifecycle:
        postStart:

Reply:
I think what happens in that case is, ovs binaries are present on host and path is added at machine level, so antrea-agent container can run ovs commands as antrea-agent is running as a `hostProcess: true` ( Reference : antrea-windows-containerd.yml)

Reply:
I think, if we would have had same image for antrea-agent and antrea-ovs container, then this issue wouldn't be there.

Reply:
Does ovs ctl have many dependencies? Can we install the ctls to antrea-windows image, instead of depending on the binaries in other image and introducing several complexities: start order dependency, difference in antrea-agent container scripts.

Reply:
There is one more dependency for ovs commands to work, .dll files from openssl which is inside antrea-ovs container at `C:\Windows\System32\`. This path is also updated and fetched just like ovs binaries file path. https://github.com/antrea-io/antrea/blob/c372130ca6a622f9e0e522792b8543d3db15d951/build/images/ovs/Dockerfile.windows#L39

Reply:
ctls can be included in antrea-windows image, however we created antrea-ovs image to cater to regular ovs update which might not be case for antrea-windows, if we add ctls to antrea-windows, with each ovs release update, we would need to update antrea-ovs image and antrea-windows image.

Reply:
@tnqn  @XinShuYang  @rajnkamr I copied OVS files inside antrea-agent container and ran ovs commands inside antrea-agent container, it exits silently without any output which is the case when dependencies are unavailable.

Reply:
@Atish-iaf Did you copy dll file to agent image and set path in agent container like what we did in line 41?

Reply:
openssl dll files copied to `windows/system32/` and vc dll files not copied. Path was set.

Reply:
antrea-windows image change is included in this PR, PTAL.
Now, there is only 1 line difference that is path update in antrea agent cm, we can have same cm across all windows yaml if we can add this path update line. Do you see any issue if we add this line or we should keep separate cm with this change ? @tnqn